scheduler/services/visibility/calculator.py

```diff
@@ -169,8 +167,17 @@ async def calculate(self) -> None:
                     _logger.info(f'Visibility calcs for {semester} from Redis retrieved.')
 
         else:
-            # fill the table manually. see fill_redis code.
-            raise NotImplementedError('Manual population not yet available')
+            from scheduler.scripts.fill_redis import compute_visibility, transform_to_json
```

Comment:
The problem with this is that we are creating two collectors, one is run in the script so we can parsed the observations and the other is the main one. I think is not a bad compromise but in this case than lets move both compute_visibility to the visibility calculator service and then transform_to_json to the redis client. Calling from the script does not look clean, the script should be that a one time interaction that ends not a module itself.

Reply:
Sure, I agree, I just started doing the change in the `fill_redis` script to keep the previous logics, actually is pretty much a refactor of the script code.